src/citations.jl

```diff
@@ -5,32 +5,43 @@ function citation_path(pkg)
     end
 end
 
-function get_citation(pkg)
+
+# Added `zenodo` flag to avoid breaking current tests
+function get_citation(pkg; zenodo=false)
```

Comment:
I think that it might be better to have a more general flag like `badge` (or `include_badges` or something like that).